com.unity.robotics.urdf-importer/Runtime/UrdfComponents/UrdfJoints/UrdfJointPrismatic.cs

```diff
@@ -20,34 +20,14 @@ namespace Unity.Robotics.UrdfImporter
     public class UrdfJointPrismatic : UrdfJoint
     {
         private ArticulationDrive drive;
-#if UNITY_2020_1
-        private float maxLinearVelocity;
```

Comment:
Was this not being used anywhere ?

Reply:
I think that we didn't have the max linear velocity property in the initial release of ABs back in 2020.1. It appears later, in 2020.2. I'm cutting this compatibility layer because 2020.1 is an obsolete tech stream pre-empted by 2020.3 LTS. No customer should be using it, and we don't need to support it any more.